apps/flows/src/index.html

```diff
@@ -6,6 +6,18 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type=”image/svg+xml” href="favicon.svg" />
+
+    <meta name="color-scheme" content="light dark" />
```

Comment:
Is `StatusBar.setStyle({})` necessary if the color scheme here is only light?

Reply:
I tried to set the content to only "light", but that didn't have an effect 🤔